microservice/src/main/java/com/skytouch/microservice/model/ProductDB.java

```diff
@@ -0,0 +1,32 @@
+package com.skytouch.microservice.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Data
+@Entity
+@Table(name = "products")
+@NamedStoredProcedureQueries({
+        @NamedStoredProcedureQuery(name = "read_products", procedureName = "read_products")
+})
+public class ProductDB {
+    @Id
+    @Column
+    private String id;
+    @Column
+    private String name;
+    @Column
+    private String description;
+    @Column
+    private BigDecimal price;
+    @Column
```

Comment:
It seems that the `name` value is missing:

 ```suggestion
     @Column(name = "is_deleted")
```